frontend/src/components/modals/modalparts/TeamMemberModalParts.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react';
+import { Trans } from 'react-i18next';
+import classNames from 'classnames';
+import { RadioButton } from '../../forms/RadioButton';
+import { RoleType } from '../../../../../shared/types/customTypes';
+import css from './TeamMemberModalParts.module.scss';
+
+const classes = classNames.bind(css);
+
+export const SelectMemberRole: FC<{
+  role: string;
+  onChange: (role: string) => void;
+}> = ({ role, onChange }) => (
```

Comment:
Using the RoleType values could simplify this

```suggestion
export const SelectMemberRole: FC<{
  role: RoleType;
  onChange: (role: RoleType) => void;
}> = ({ role, onChange }) => (
```